src/bin/add/fetch.rs

```diff
@@ -36,20 +37,30 @@ pub fn get_latest_dependency(crate_name: &str) -> Result<Dependency, FetchVersio
     Ok(dep)
 }
 
+// Checks whether a version object is a stable release
+fn version_is_stable(version: &Object) -> bool {
+    version.get("num")
+        .and_then(Json::as_string)
+        .and_then(|s| Version::parse(s).ok())
```

Comment:
Is the `ok()` necessary? You should be able to call `map` and `unwrap_or` on Result as well.